auslib/web/admin/swagger/api.yaml

```diff
@@ -2345,22 +3372,75 @@ definitions:
                 version: null
                 whitelist: null
 
-  RulesBase:
-    title: Rules object definition
-    description: Balrog's Rule Attributes
+  UpdateType:
+    title: Rule's update_type field
+    description: Non-Null field of Rule to specify type of update
     type: object
+    properties:
+      update_type:
+        description: "The update_type to use in the XML response. 'minor' is almost always the correct value these days."
+        type: string
+        minLength: 0
+        maxLength: 15
+        enum: [ "major", "minor"]
+        example: minor
 
-    # Cannot have additionalProperties as false for strict validation ,else it disallows csrf_token or any other model's properties to be added
-    #additionalProperties: false
+  RulesNonNullable:
```

Comment:
No that's not correct. Some fields aren't always required. Seems like a misnomer. Also priority and backgroundRate are actually nullable in many cases when handling request/response in /rules APIs. 

Any name that can automatically signify that model's fields can be of any type but 'null' will do here